src/ace/managers/ptplayer.c

```diff
@@ -2533,15 +2536,15 @@ static void set_toneporta(
 	// Find first period which is less or equal the note in d6
 	UWORD uwNote = pVoice->uwNote & 0xFFF;
 	UBYTE ubPeriodPos = findPeriod(mt_PeriodTables[0], uwNote);
-	if(ubPeriodPos) {
-		// One before for less/equal
-		--ubPeriodPos;
-	}
-
-	if(pChannelData->n_minusft && ubPeriodPos) {
-		// Negative fine tune? Then take the previous period.
-		--ubPeriodPos;
-	}
+	// if(ubPeriodPos) {
+	// 	// One before for less/equal
+	// 	--ubPeriodPos;
+	// }
+
+	// if(pChannelData->n_minusft && ubPeriodPos) {
+	// 	// Negative fine tune? Then take the previous period.
+	// 	--ubPeriodPos;
+	// }
```

Comment:
what's going on here?

Reply:
from @rveilleux 's commit message:
> this change is not 100% certain because the original ASM code does something similar, but since it does cmp.w (a0)+,d6, this always increment a0 one element past where it should.
> Might need revist, but at least now it sounds accurate and not out-of-tune